src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int ZERO_VALUE = 0;
+    private static final int SECOND_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        writeToFilee(doNeededForm(getSumValue(fromFileName)),toFileName);
+    }
+
+    private String readFromFile(String adres) {
+        File file = new File(adres);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        String result = stringBuilder.toString();
+        return result;
+    }
```

Comment:
```suggestion
    private String readFromFile(String fileName) {
        File file = new File(fileName);
        StringBuilder stringBuilder = new StringBuilder();
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
            String value = bufferedReader.readLine();
            while (value != null) {
                stringBuilder.append(value).append(System.lineSeparator());
                value = bufferedReader.readLine();
            }
        } catch (IOException e) {
            throw new RuntimeException("Can't read from file: " + fileName, e);
        }
        return stringBuilder.toString();
    }
```
change namings
make exceptions more informative
remove redundant local variable